Library/Formula/json-fortran.rb

```diff
@@ -0,0 +1,73 @@
+class JsonFortran < Formula
+  desc "A Fortran 2008 JSON API"
+  homepage "https://github.com/jacobwilliams/json-fortran"
+  url "https://github.com/jacobwilliams/json-fortran/archive/4.1.1.tar.gz"
+  sha256 "97f258d28536035ef70e9ead5c7053e654106760a12db2cc652587ed61b76124"
+
+  head "https://github.com/jacobwilliams/json-fortran.git"
+
+  option "with-unicode-support", "Build json-fortran to support unicode text in json objects and files"
+  option "with-keg-only", "Keg only install (Skip linking json-fortran.) Required to support multiple Fortran compilers."
+  option "without-make-test", "Skip running build-time tests (not recommended)"
+
+  depends_on "cmake" => :build
```

Comment:
OK, question though: I need CMake for `brew test json-fortran` because without it the test fails AND I need it to build json-fortran but it is not a true runtime dependency. What is the correct way to specify the `:build` dependency and the dependency for `test do …`? 

I originally had only `depends_on “cmake” => :build` because cmake is needed to build the project but the test was failing when testbot ran it. (`brew test json-fortran` was working on my local machine, I guess since I have cmake installed and on my path…) It appears adding this second line fixed the testbot failure.